pom.xml

```diff
@@ -315,6 +315,12 @@
       <scope>provided</scope>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>io.github.skytasul</groupId>
+      <artifactId>glowingentities</artifactId>
```

Comment:
Is this an API that need to be updated with every Minecraft version? Then I think we can not accept this pr, as we do not build new BQ versions for new MC versions.